h2o-r/h2o-package/R/kvstore.R

```diff
@@ -287,10 +287,46 @@ h2o.getModel <- function(model_id) {
   }
 }
 
+#' Retrieves an instance of \linkS4class{H2OSegmentModels} for a given id.
+#'
+#' @param segment_models_id A string indicating the unique segment_models_id
+#         of the collections of segment-models to retrieve.
+#' @return Returns an object that is a subclass of \linkS4class{H2OSegmentModels}.
+#' @examples
+#' \dontrun{
+#' library(h2o)
+#' h2o.init()
+#' iris_hf <- as.h2o(iris)
+#' h2o.segment_train(algorithm = "gbm",
+#'                   segment_columns = "Species", segment_models_id="models_by_species",
+#'                   x = c(1:3), y = 4, training_frame = iris_hf, ntrees = 5, max_depth = 4)
+#' models <- h2o.getSegmentModels("models_by_species")
+#' as.data.frame(models)
+#' }
+#' @export
 h2o.getSegmentModels <- function(segment_models_id) {
```

Comment:
```suggestion
h2o.get_segment_models <- function(segment_models_id) {
```

Reply:
In the last year or more, we have dropped the camel case for `h2o.get*` functions in R and switched to snake case for all new functions (we are aspiring to modern R coding conventions), so I would suggest we do that here too.  At some point it might just make sense to make aliases for the remainder of the functions that still use camel case like `h2o.getModel()`, `h2o.getFrame()`, etc. but that's a discussion for another day!  We have at least one alias, `h2o.getAutoML()` (created three years ago) and now we also have `h2o.get_automl()`.

Reply:
sounds good, I was previously following the getModel pattern for consistency